ratpack-core/src/main/java/ratpack/server/internal/Slf4jNoBindingDetector.java

```diff
@@ -36,7 +36,8 @@ private static boolean init() {
     } catch (ClassNotFoundException e) {
       System.err.println(
         "WARNING: No slf4j logging binding found for Ratpack, there will be no logging output." + System.lineSeparator()
-          + "WARNING: Please add an slf4j binding, such as slf4j-log4j2, to the classpath."
+          + "WARNING: Please add an slf4j binding, such as slf4j-log4j2, to the classpath." + System.lineSeparator()
+          + "WARNING: More info may be found here: http://ratpack.io/manual/current/logging.html"
```

Comment:
For those reasons yes, but slf4j-simple is blocking so I’d rather not promote it here.